assets/css/darktheme.css

```diff
@@ -1,9 +1,9 @@
-a:hover,
-a:active {
-  color: rgb(0, 182, 240);
+a:not(.pure-button):hover,
+a:not(.pure-button):active {
+  color: rgb(0, 182, 240) !important;
```

Comment:
This was done to stay consistent with `lighttheme.css`. Sorry for putting the change it in this commit / PR.